app/styles/partials/footer.scss

```diff
@@ -24,15 +23,16 @@ footer {
   padding-top: 30px !important;
 
   .language.menu > div {
-    color: rgba(0, 0, 0, .87) !important;
+    color: rgba(0, 0, 0, 0.87) !important;
```

Comment:
Why

Reply:
These changes occurred automatically when I made a commit after changing the .scss file - Error message"We've failed to pass the specified git pre-commit hooks as the `lint` pre-commit: hook returned an exit code (1)".

It further said that it cannot be 0.

Please help me solve this issue.